### PR TITLE
Crncc 968 areas of research formatting

### PR DESCRIPTION
**Pull request by tanishawPA**
Button's text was made uppercase by MUI button components. Added modifier to remove uppercase from the selected areas of health interest buttons.

### CODE REVIEW

**Review by paddyduff31 — CHANGES_REQUESTED** [review]
This change says nothing about Button casing, this is to do with the casing of the selected areas of research by the users.  Need to re-read the requirements:
https://nihr.atlassian.net/browse/CRNCC-968

File: src/components/Shared/UI/DTECheckList/DTECheckList.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ const DTECheckList = memo(({ id, name, label, hint, error, values, onValueChange
       disabled={item.disabled}
       conditional={item.conditional}
       onClick={(e: any) => handleOnChange(e)}
-      onKeyPress={(e: any) => handleOnChange(e)}
+      // onKeyPress={(e: any) => handleOnChange(e)}
```
Comment (paddyduff31) [inline src/components/Shared/UI/DTECheckList/DTECheckList.tsx:68]: @tanishawPA why has this been commented out?

Comment (tanishawPA) [reply src/components/Shared/UI/DTECheckList/DTECheckList.tsx:68]: The line was greyed out and said it was a deprecated function. I can go back and just remove the line? Wasn't super sure!

**Comment by tanishawPA** [comment]
> This change says nothing about Button casing, this is to do with the casing of the selected areas of research by the users. Need to re-read the requirements:

Changes discussed, and happy to proceed!

File: src/components/Shared/UI/DTECheckList/DTECheckList.tsx (hunk 1 above)
Comment (paddyduff31) [reply src/components/Shared/UI/DTECheckList/DTECheckList.tsx:68]: @tanishawPA I have had a look and it's not being used in the application.  Just be sure if you are removing/commenting anything out you have a reason and understand why.  It looks like it is fine to remove so feel free to delete but if future make sure you understand the "why".  

Nothing should make it to github if you don't understand, as who knows what you could be affecting!

**Review by paddyduff31 — APPROVED** [review]

**Pull request merged**